internal/generator/examples/default/simple.en-pirate.feature_test.go

```diff
@@ -0,0 +1,25 @@
+package examples_test
+
+import (
+	"testing"
+
+	"github.com/hedhyw/gherkingen/v3/pkg/bdd"
+)
+
+func TestGuessinTheWord(t *testing.T) {
+	t.Parallel()
+
+	f := bdd.NewFeature(t, "Guessin' the Word!")
+
+	/*
+		This be a game o' guessin' words, turn-based and for two scallywags only!
+		The Maker, a cunning buccaneer, chooses a secret word for the Breaker to decipher.
+		The Breaker, a sharp wit, tries to guess the Maker's hidden word.
+		The game be done when the Breaker shouts out the right word!
+	*/
+
+	f.HeaveTo("The Maker starts a game!", func(t *testing.T, f *bdd.Feature) {
```

Comment:
`HeaveTo` is not implemented by the (bdd)[https://github.com/hedhyw/gherkingen/tree/v3.0.3/pkg/bdd] package. Which breaks the tests. I am not sure wherther it is worth implementing language-specific method names for all supported natural languages or not. There are quite a lot of them now, and later, perhaps, there will be even more.

From my point of vies, it is useful to have a feature written in the natural language. While having tests for this feature written in natural language inside a programming language is not. Therefore, I propose to redesigh the templates so that they will not deviate from the well-known English keywords [Given, When, Then...](https://cucumber.io/docs/gherkin/reference/) regardless of the natural language choosen for the feature itself.

Reply:
> From my point of view, it is useful to have a feature written in the natural language. While having tests for this feature written in natural language inside a programming language is not. Therefore, I propose to redesign the templates so that they will not deviate from the well-known English keywords [Given, When, Then...](https://cucumber.io/docs/gherkin/reference/) regardless of the natural language choosen for the feature itself.

note: Agree! Actually this "bdd" package is deprecated. We can remove later and bump major version.

Reply:
> We can remove later and bump major version.
Do you mean not in this Pull Request, or when?

Reply:
Currently, we can just remove from examples

Reply:
I've removed it in https://github.com/hedhyw/gherkingen/pull/122